docs/ref/utils.txt

```diff
@@ -155,13 +155,21 @@ The functions defined in this module share the following properties:
 
     Converts a function decorator into a method decorator. It can be used to
     decorate methods or classes; in the latter case, ``name`` is the name
-    of the method to be decorated and is required. See :ref:`decorating
+    of the method to be decorated and is required.
+    If ``decorator`` is an iterable of decorators, the aforementioned behavior
+    is repeated for each decorator yielded by the iterable.
+    See :ref:`decorating
     class based views<decorating-class-based-views>` for example usage.
 
     .. versionchanged:: 1.9
 
        The ability to decorate classes and the ``name`` parameter were added.
 
```

Comment:
don't need a separate `.. versionchanged:: 1.9`
"Also, the ability to use an iterable for the `decorator` was added."